exporters/prometheus/exporter.go

```diff
@@ -232,6 +245,55 @@ func getAttrs(attrs attribute.Set) ([]string, []string) {
 	return keys, values
 }
 
+func createTargetInfoMetric(res *resource.Resource) (*metricData, prometheus.Labels) {
+	if res == nil {
+		return nil, nil
+	}
+
+	var namespace, name, instance string
```

Comment:
Mapping service resource attributes to job/instance [should only(https://github.com/open-telemetry/opentelemetry-specification/blob/v1.14.0/specification/metrics/data-model.md#resource-attributes-1) be done in collector prometheus exporters.

Reply:
I've missed the point that this is specific for the collector. Thanks for noting this!

Updated the code for not adding job/instance labels.